submissions/masturohAl-Ameen/HTML/productpage.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Product Page</title>
+</head>
+<body>
+    <nav>
+        <a href="productpage.html"><b>INNER GLAMOUR</b></a>
+    </nav>
+    <h4>Welcome to Inner Glamour, Your Secret Specials!</h4>
+
+    <!--This section contains products, their prices and description-->
+    <section>
+        <h2>Leggings</h2>
+        <img src="images/leggings.jpg" alt="leggings">
+        <p> &#x20A6;20,000 </p>
+        <h3>Description</h3>
+        <p>This is an exquisite leggings made from excellent material.<br>It is durable and smooth on the skin. It can also be used for various activities.</p> 
+    </section>
+    
+    <hr>
+
+    <section>
+        <h2>Shorts</h2>
+        <img src="images/shorts.jpg" alt="shorts">
+        <p> &#x20A6;15,000 </p>
+        <h3>Description</h3>
+        <p>This shorts are great for indoors activities.<br>You can also sleep in it when you feel like not wearing that night gown.</p>
+    </section>
+
+    <section>
+        <h2>Other Available Products</h2>
+        <ol>
+            <li>Pants</li>
+            <li>Brassiers</li>
+            <li>Body Con</li>
+            <li>Lingerie</li>
+            <li>Camisoles</li>
+        </ol>
+    </section>
+
```

Comment:
It's good you try to be consistent with formatting. You did not leave a line before opening the body tag but I can see you are doing that for the footer tag.